eng/common/TestResources/New-TestResources.ps1

```diff
@@ -546,12 +560,28 @@ try {
             }
 
             $outputFile = "$($templateFile.originalFilePath).env"
+            if ($env:ENV_FILE) {
+                $outputFile = "$env:ENV_FILE"
+            }
 
-            $environmentText = $deploymentOutputs | ConvertTo-Json;
-            $bytes = ([System.Text.Encoding]::UTF8).GetBytes($environmentText)
-            $protectedBytes = [Security.Cryptography.ProtectedData]::Protect($bytes, $null, [Security.Cryptography.DataProtectionScope]::CurrentUser)
+            $environmentText = ''
+            if ($OutFileFormat -eq 'json') {
+                $environmentText = $deploymentOutputs | ConvertTo-Json;
+            } elseif ($OutFileFormat -eq 'dotenv') {
+                foreach($entry in $deploymentOutputs.GetEnumerator()) {
```

Comment:
Are these the same output with just a different file extension?

Reply:
No, the outputs are in different formats
json outputs
{
  <key>:<val>
}
dotenv outputs
<key>=<val>
pwshenv outputs
$<key>=<val>

Reply:
1. JSON output is for existing code and scripts
2. dotenv output is to support stress tests
3. I figured "pwshenv" would be useful to add here so people can more easily dot source the env, but this might be more complicated with encrypted files.